orchagent/portsorch.cpp

```diff
@@ -5307,9 +5339,14 @@ bool PortsOrch::initializePort(Port &port)
             {
                 operStatus = fvValue(i);
             }
+
+            if (fvField(i) == "flap_count")
```

Comment:
Is this to handle reconciliation during warm/fast reboot?

Reply:
@dgsudharsan yes